packages/react-composites/review/react-composites.api.md

```diff
@@ -289,7 +289,13 @@ export type CallIdChangedListener = (event: {
 // Warning: (ae-incompatible-release-tags) The symbol "ChatAdapter" is marked as @public, but its signature references "FileUploadAdapter" which is marked as @beta
 //
 // @public
+<<<<<<< HEAD
```

Comment:
This shouldn't be here.
You might need to regenerate the api.md files by running
`rush build -v`

Reply:
done